test cases/cmake/7 cmake options/subprojects/cmOpts/CMakeLists.txt

```diff
@@ -5,6 +5,6 @@ if(NOT "${SOME_CMAKE_VAR}" STREQUAL "something")
   message(FATAL_ERROR "Setting the CMake var failed")
 endif()
 
-if(NOT "${CMAKE_PREFIX_PATH}" STREQUAL "val1;val2")
+if(NOT "${CMAKE_PREFIX_PATH}" STREQUAL "/some/prefix/path")
```

Comment:
We really should keep testing for list conversion/handling, so please revert that change.

Reply:
I dropped that change, and adjusted the schema to allow `val:` and `exclude:` to contain arrays of strings as well.